src/ShopBundle/objects/WebModules/MTShopArticleCatalogCore/MTShopArticleCatalogCoreEndPoint.class.php

```diff
@@ -381,10 +388,17 @@ public function WriteReview()
                 $iActiveCategoryId = $oActiveCategory->id;
             }
             $sURL = $oActiveItm->GetDetailLink(true, $iActiveCategoryId);
+            /**
+             * @psalm-suppress UndefinedInterfaceMethod
+             * @FIXME `HeaderURLRedirect` only exists on a single interface implementation.
```

Comment:
`HeaderURLRedirect` only exists on a single interface implementation, not on the interface itself. This came up a couple of times - but I'll only add a comment here once.